tests/integrate/tools/sum_ENV_H2.cpp

```diff
@@ -0,0 +1,55 @@
+#include <string>
+#include <iostream>
+#include <sstream>
+#include <fstream>
+#include <iomanip>
+#include <cmath>
+
+using namespace std;
+
+int main(int argc, char **argv)
+{
+	string input_file = argv[1];
+
+	ifstream inp(input_file.c_str(), ios::in);
+
+	if (!inp) 
+	{
+        	cout << "\e[1;31m [  FAILED  ]  \e[0m";
+		cout << "Can't find " << input_file << " !" << endl;
+		return 0;
```

Comment:
For linux convention, programs returns 0 for normal exit, and 1 for abnormal ones.

Reply:
OK